common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4075,6 +4075,9 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "If hive (in tez mode only) cannot find a usable hive jar in \"hive.jar.directory\", \n" +
         "it will upload the hive jar to \"hive.user.install.directory/user.name\"\n" +
         "and use it to run queries."),
+    HIVE_MASKING_ALGO("hive.masking.algo","sha256", "This property is used to indicate whether " +
```

Comment:
Why do we need a new property, could we just change the algorithm to sha512?

Reply:
It is not backward compatible.

Reply:
Did you run tests? Could you add a test for it?